### DISCUSSION TITLE
Wrong name "Maximum upload size" in the "System – Configuration" section

### DISCUSSION THREAD

**Discussion by alphagamma5** [YetiForceCompany/YetiForceCRM#15430] [Feature improvement]
Wrong name "Maximum upload size" in the "System – Configuration" section,
in fact, there is a variable post_max_size, that is, the maximum size of the post.

Неправильное название "Maximum upload size" в разделе "System – Configuration",
на самом деле там переменная post_max_size, то есть Максимальный размер поста.

**Comment by vovpff** [comment]
Этот параметр относится с POST/GET запросам к вебсерверу, по этому `Maximum upload size` название параметра как раз то что нужно

**Comment by mariuszkrzaczkowski** [comment]
the name is correct, both parameters should be set the same
https://www.a2hosting.com/kb/developer-corner/php/using-php.ini-directives/php-maximum-upload-file-size
